infrastructure-playbooks/cephadm.yml

```diff
@@ -219,11 +219,17 @@
       environment:
         CEPHADM_IMAGE: '{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'
 
-    - name: allow cephadm key for root account
+    - name: allow cephadm key for {{ cephadm_ssh_user | default('root') }} account
       authorized_key:
-        user: root
+        user: "{{ cephadm_ssh_user | default('root') }}"
```

Comment:
so this variable must be passed as an extra var?

Reply:
yes we do the same in #6768
Want to move both to ceph-defaults?